src/components/layers/drawing/featuresCounter.jsx

```diff
@@ -0,0 +1,19 @@
+
+
+const featuresCounter = (layers, geometry) => {
+    const counter = new Array(layers.length).fill(0)
+    const type = geometry.getType();
+    if (type === "Polygon") {
+      layers.forEach((layer,index) =>
+      layer.getSource().forEachFeature(function(feature){

```

Comment:
Please adhere to using `const` as a convention rather than using `function`, unless hoisting is needed